modules/connectors/ninja_van/karrio/mappers/ninja_van/settings.py

```diff
@@ -9,11 +9,14 @@ class Settings(provider_utils.Settings):
     """Ninja Van connection settings."""
 
     # required carrier specific properties
+    client_id: str = None
+    client_secret: str = None
+    grant_type: str = "client_credentials"
 
     # generic properties
     id: str = None
     test_mode: bool = False
     carrier_id: str = "ninja_van"
-    account_country_code: str = None
+    country_code: str = "SG"
```

Comment:
Please keep it as it is because it's a generic property
```suggestion
    country_code: str = "SG"
```
and use `account_country_code` as `country_code`
```suggestion
    account_country_code: str = "SG"
```

Reply:
Done